### PR TITLE
don't update symbols on editor switching

### PR DESCRIPTION
**Pull request by aviatesk**
fixes https://github.com/JunoLab/Juno.jl/issues/407
ref: https://github.com/JunoLab/Atom.jl/pull/205

### CODE REVIEW

File: lib/runtime/outline.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export function activate (ink) {
   subs.add(client.onDetached(() => pane.setItems([])))
 
   let edSubs = new CompositeDisposable()
-  subs.add(atom.workspace.onDidChangeActiveTextEditor(ed => {
+  subs.add(atom.workspace.onDidChangeActiveTextEditor(throttle(ed => {
```
Comment (aviatesk) [inline lib/runtime/outline.js]: I think `throttle` here would be reasonable, and won't make Julia hard-work when consecutive editor switches.

Comment (pfitzseb) [reply lib/runtime/outline.js]: Yeah, good call.

File: lib/runtime/outline.js [hunk 2]
```diff
@@ -60,14 +60,18 @@ export function activate (ink) {
       pane.setItems(outline)
     }, 300)))
 
-    updateEditor(ed).then(outlineItems => {
+    updateEditor(ed, {
+      noUpdateSymbols: true
```
Comment (pfitzseb) [inline lib/runtime/outline.js]: Nitpick, but I feel like `updateSymols: false` would be easier to parse (here and in the Julia code).

**Comment by aviatesk** [comment]
good to go.

**Pull request merged**